### ISSUE TITLE
Mana pot and consume usage of the person keeping the log

### ISSUE THREAD

**Issue by Bombg** [pepopo978/SuperWowCombatLogger#6]
Took a log of my guild's BWL run tonight and ran the log through formatter for uploading to monkeylogs. So far that seems to be working well.

But after using the summarize_consumes tool on the formatted log, I noticed it didn't show any mana pot usage for myself. It did show most of my other consume usage, but also didn't show the second wisdom flask I used as well. although thinking about it, maybe the 2nd usage didn't show up because I popped the 2nd flask before the first wore off.

Here's a gist of the raw log pre-formatter if you wish to use it for testing.

https://gist.githubusercontent.com/Bombg/b5c233611b3923528f452cc003f17b2a/raw/b84e32ffff5bdabf5c79f14c20cc2819c842883d/gistfile1.txt

**Comment by pepopo978** [comment]
Hiya what exactly is the summarize_consumes tool?  I think I've seen the output of it but don't know where to get it.  If it's the tool I'm thinking of I have seen it miss consumes before.

This is how mana pots appear in the log, doesn't seem to have changed:
old log `4/19 21:12:25.950  Friendhelper gains 1759 Mana from Friendhelper 's Restore Mana.`
new log `4/14 20:38:21.794  Ehawne gains 2202 Mana from Ehawne 's Restore Mana.`

**Comment by pepopo978** [comment]
think i found it https://github.com/melbaa/summarize_consumes

**Comment by pepopo978** [comment]
Found the issue:
```
new log: `4/19 20:04:01.586  Pepopo gains 1413 Mana from Restore Mana.`
old log:  `2/16 20:21:34.043  Pepopo gains 1359 Mana from Pepopo 's Restore Mana.`
```

I'll fix when I get back later

**Comment by pepopo978** [comment]
Fixed by 85093016368ee0a27e2c9626b855ff9402a5863f
